docs/rules/no-use-before-define.md

```diff
@@ -90,25 +92,29 @@ f();
 function f() {}
 ```
 
-The following patterns are not considered problems when `"classes": false` is specified:
+### classes
```

Comment:
Good eyes: the preceding **functions** heading should have been level 3 parallel to **classes** here.